toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -179,6 +179,22 @@ SystemConfig:
       Permissions: "664"
 ```
 
+## KernelCommandLine type
+
+Options for configuring the kernel.
+
+### ExtraCommandLine
```

Comment:
should we rename that to something more obvious. Also It seems to me that this should move under the partition API

Reply:
I am trying to remain consistent with the imager's config.